### ISSUE TITLE
Add the extended Environment Indicator module

### ISSUE THREAD

**Issue by NicoledeGreef** [bcgov/bcbb#45]
Some time ago, the existence of an extended env variables module (coloured bar?) was discussed for addition to the BCBB but it was not ticketized. Creating a ticket now as a reminder in case that is something that the devs want to add to the bcbb to benefit future users.

**Comment by danhgov** [comment]
This is not going to work as-is, because our other projects that I built this for use the Gin toolbar.

However, I just made a commit (on the mfin-data-catalog repo) to include the environment-switcher config for this project.

**Comment by danhgov** [comment]
@NicoledeGreef, I don't think we should be proceeding with this ticket, putting our enhancements to the Environment Indicator into BCBB, because, as I mentioned above, this project is not using the Gin toolbar, like our other projects are (but maybe we could switch it?)

However, I've created https://github.com/bcgov/MFIN-Data-Catalogue/issues/496 to nonetheless make some improvements to the Environment Indicator on MFIN-DC.
